<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <comment>****************************************************************************
**
** Copyright (C) 2013-2025 Donna Whisnant, a.k.a. Dewtronics.
** Contact: http://www.dewtronics.com/
**
** This file is part of the KJVCanOpener Application as originally written
** and developed for Bethel Church, Festus, MO.
**
** GNU General Public License Usage
** This file may be used under the terms of the GNU General Public License
** version 3.0 as published by the Free Software Foundation and appearing
** in the file gpl-3.0.txt included in the packaging of this file. Please
** review the following information to ensure the GNU General Public License
** version 3.0 requirements will be met:
** http://www.gnu.org/copyleft/gpl.html.
**
** Other Usage
** Alternatively, this file may be used in accordance with the terms and
** conditions contained in a signed written agreement between you and
** Dewtronics.
**
****************************************************************************</comment>
 <class>CCrossRefEditDlg</class>
 <widget class="QDialog" name="CCrossRefEditDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>592</width>
    <height>465</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>King James Pure Bible Search Cross Reference Editor</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="childrenCollapsible">
      <bool>false</bool>
     </property>
     <widget class="QWidget" name="layoutWidgetSource">
      <layout class="QVBoxLayout" name="verticalLayoutSource">
       <item>
        <widget class="QLabel" name="labelSourcePassage">
         <property name="text">
          <string>Source &amp;Passage:</string>
         </property>
         <property name="buddy">
          <cstring>editSourcePassage</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="editSourceRefDesc">
         <property name="frameShape">
          <enum>QFrame::Panel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Sunken</enum>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="margin">
          <number>4</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextEdit" name="editSourcePassage">
         <property name="sizePolicy">
          <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
           <horstretch>10</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>150</height>
          </size>
         </property>
         <property name="mouseTracking">
          <bool>true</bool>
         </property>
         <property name="acceptDrops">
          <bool>false</bool>
         </property>
         <property name="tabChangesFocus">
          <bool>true</bool>
         </property>
         <property name="undoRedoEnabled">
          <bool>false</bool>
         </property>
         <property name="textInteractionFlags">
          <set>Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="buttonSelectSourceRef">
         <property name="text">
          <string>Select &amp;Source Reference...</string>
         </property>
         <property name="icon">
          <iconset resource="KJVCanOpener.qrc">
           <normaloff>:/res/Nav3_Arrow_Up.png</normaloff>:/res/Nav3_Arrow_Up.png</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidgetRefList">
      <layout class="QVBoxLayout" name="verticalLayoutRefList">
       <item>
        <widget class="QLabel" name="labelCrossRefs">
         <property name="text">
          <string>&amp;Cross References:</string>
         </property>
         <property name="buddy">
          <cstring>treeCrossRefs</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTreeView" name="treeCrossRefs"/>
       </item>
       <item>
        <widget class="QPushButton" name="buttonAddRef">
         <property name="text">
          <string>&amp;Add Reference...</string>
         </property>
         <property name="icon">
          <iconset resource="KJVCanOpener.qrc">
           <normaloff>:/res/green_arrow.png</normaloff>:/res/green_arrow.png</iconset>
         </property>
         <property name="shortcut">
          <string>Ins</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="buttonDeleteRef">
         <property name="text">
          <string>&amp;Delete Reference(s)</string>
         </property>
         <property name="icon">
          <iconset resource="KJVCanOpener.qrc">
           <normaloff>:/res/deletered1-24.png</normaloff>:/res/deletered1-24.png</iconset>
         </property>
         <property name="shortcut">
          <string>Del</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>editSourcePassage</tabstop>
  <tabstop>treeCrossRefs</tabstop>
  <tabstop>buttonAddRef</tabstop>
  <tabstop>buttonDeleteRef</tabstop>
  <tabstop>buttonSelectSourceRef</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources>
  <include location="KJVCanOpener.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>CCrossRefEditDlg</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>CCrossRefEditDlg</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
